src/Umbraco.Web.BackOffice/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -47,7 +47,8 @@ public static class UmbracoBuilderExtensions
                 .AddPreviewSupport()
                 .AddHostedServices()
                 .AddDistributedCache()
-                .AddModelsBuilderDashboard();
+                .AddModelsBuilderDashboard()
+                .AddUnattedInstallCreateUser(); // Put last to test that everything else injected/setup & happy
```

Comment:
Please remove that comment.  The ordering of composing should not be important, because all composing happen before the actual pipeline boot.